### PR TITLE
Don't reload timeline page when (un)resolving or replying conversation

### LINKED ISSUE #15981: [Bug] Do not reload page after resolving a conversation
- Gitea version (or commit ref): 1.15.0+dev-344-g5285a3e70
- Can you reproduce the bug at https://try.gitea.io:
  - Yes: Feel free to experiment [here](https://try.gitea.io/delvh/kanban-test/pulls/10) yourself

## Description
Currently, when resolving a conversation in a PR inside the `Conversation` tab, the page automatically gets reloaded.
Surprisingly, when resolving a conversation there inside the `Files Changed` tab, then it does not get reloaded.
This is interesting as I would have expected both to use the same exact code, given that both should do exactly the same.
Would it be possible to not have to endure a reload every time a conversation is resolved in the `Conversation` tab, simply by replacing the currently present code with the one used in the `Files changed` tab?

### PR DESCRIPTION
**Pull request by jpraet**
Fixes #15981

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
Last Call @go-gitea/technical-oversight-committee

**Review by denyskon — APPROVED** [review]

**Comment by delvh** [comment]
Erm… What is our current stance on htmx?
I've only heard the debate peripherally, did we revert it or not for now?
If we didn't, wouldn't it make sense here to use htmx as well?

**Comment by denyskon** [comment]
We did revert, without a consensus on how to proceed yet. I think we can merge this, and reimplement in the future if needed. After all, this PR doesn't add much code.

File: templates/repo/issue/view_content/conversation.tmpl [hunk 1]
```diff
@@ -0,0 +1,133 @@
+{{$invalid := (index .comments 0).Invalidated}}
+{{$resolved := (index .comments 0).IsResolved}}
+{{$resolveDoer := (index .comments 0).ResolveDoer}}
+{{$isNotPending := (not (eq (index .comments 0).Review.Type 0))}}
```
Comment (wxiaoguang) [inline templates/repo/issue/view_content/conversation.tmpl:4]: ->  Do not render empty comments #29039

**Pull request merged**